docker-compose.yml

```diff
@@ -152,6 +152,25 @@ services:
       - 18222
     networks:
       - chs-dev
+  api.ch.gov.uk:
+    image: 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/api.ch.gov.uk:latest
+    container_name: api.ch.gov.uk
+    labels:
+      - traefik.http.routers.api.ch.gov.uk.rule=Host(`chs-dev`) && PathPrefix(`/api.ch.gov.uk/`)
```

Comment:
Would the URL to access this service be `http://chs-dev/api.ch.gov.uk/...` ?

Reply:
yes.

Reply:
Should this be it's own host? `http:/api.ch.gov.uk/`? It's not a web UI

Reply:
agreed. I'll fix that.